### PR TITLE
Add details to Add-SPOListdesign to support rollout of doc lib org te…

### PR DESCRIPTION
**Pull request by reedpamsft**
…mplates

Soon we will be rolling out an update to doc lib creation that means that doc lib templates created through add-spolistdesign will flow to the correct UI (lists into list UI, doc libs into doc lib UI).  This update lets users know about the change.

### CODE REVIEW

**Comment by reedpamsft** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 1]
```diff
@@ -3,16 +3,17 @@ external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
...
 ms.reviewer:
+ms.topic: List Templates
 title: Add-SPOListDesign
 ---
 
 # Add-SPOListDesign
 
 ## SYNOPSIS
-Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+Creates a new list or document library design available to users when they create a new list or document library from Site Contents, certain site home pages, the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:16]: >Site Contents [](http://example.com/codeflow?start=118&length=13)

is this a product? if not should be just lowercase

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 2]
```diff
@@ -23,7 +24,11 @@ Add-SPOListDesign -Title <String> -SiteScripts <SPOSiteScriptPipeBind[]> [-Descr
 ```
 
 ## DESCRIPTION
-Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+Creates a new list or document library design available to users when they create a new list or document library from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+
+List designs will be available in UI where lists are created, and document library designs will be available in UI where document librraries are created.  The difference is based on the "templateType" of the site design used.  "templateType" 100 is for Lists, "templateType" 101 is for document libraries.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:29]: >librraries [](http://example.com/codeflow?start=130&length=10)

typo

Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:29]: >"templateType" [](http://example.com/codeflow?start=186&length=14)

I don't see this param in the cmdlet; where does this come from?

Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:29]: >   [](http://example.com/codeflow?start=153&length=2)

remove extra space

Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:29]: >   [](http://example.com/codeflow?start=225&length=2)

remove extra space

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 3]
```diff
@@ -23,7 +24,11 @@ Add-SPOListDesign -Title <String> -SiteScripts <SPOSiteScriptPipeBind[]> [-Descr
...
+
+Note that in early 2024, a change will be rolling out which will put document library templates into the document library creation UI and list templates into the list creation UI.  Prior to this change, all designs added through this cmdlet would have appeared in the list creation UI.  Those existing designs will not automatically get moved to document library UI.  To move them to the correct location, you'll need to remove the design and add it again.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:32]: it's already 2024, is this line still relevant? If yes, I'd recommend linking out to official documentation page for this feature in learn.microsoft.com

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md (hunk 2 above)
Comment (reedpamsft) [reply sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:29]: They're part of the xml definition of the list design.

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 4]
```diff
@@ -3,16 +3,17 @@ external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
 Module Name: Microsoft.Online.SharePoint.PowerShell
 online version: https://learn.microsoft.com/powershell/module/sharepoint-online/add-spolistdesign
 schema: 2.0.0
-author: SaladiHarini
-ms.author: hasaladi
+author: Reed Pankhurst
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:6]: >Reed Pankhurst [](http://example.com/codeflow?start=8&length=14)

I think they want this to be "reedpa-microsoft"? see this PR: [https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/603/files](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/603/files)

**Review by samkabue — APPROVED** [review]
:shipit:

**Comment by samkabue** [comment]
> Learn Build status updates of commit _[2fdf11c](https://github.com/reedpamsft/OfficeDocs-SharePoint-PowerShell/commits/2fdf11c8b59a648cc435bb74a98b18fcef8cdc40)_:
> 
> ### ⚠️ Validation status: warnings
> File	Status	Preview URL	Details
> [sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md](https://github.com/reedpamsft/OfficeDocs-SharePoint-PowerShell/blob/patch-1/sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md)	⚠️Warning	[View](https://review.learn.microsoft.com/en-us/powershell/module/sharepoint-online/add-spolistdesign?view=sharepoint-ps&branch=pr-en-us-582) (sharepoint-ps)	[Details](#user-content-94b8b4390b465826fcec016f3818aa76-d6c99295)
> ### [sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md](https://github.com/reedpamsft/OfficeDocs-SharePoint-PowerShell/blob/patch-1/sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md)
> * **Line 0, Column 0**: **[Warning: author-not-found - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/author-not-found?branch=main)]** `Invalid value for author: 'reedpa-microsoft' is not a valid GitHub ID.`
> 
> For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/3dbbfaea-2c18-2d63-2f2f-97a8fabaf65d/202404101554306282-582/BuildReport?accessString=7bffb2d2ce26a00229d4f717ef1a1b4f50f9fdb88846d3d5a42d2c794326002a).
> 
> **Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.
> 
> For any questions, please:
> 
> * Try searching the learn.microsoft.com [contributor guides](https://review.learn.microsoft.com/en-us/help/?branch=main)
> * Post your question in the [Learn support channel](https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47)

@garycentric what should be the author value for this PR?

**Pull request merged**